test/e2e/kubectl.go

```diff
@@ -202,6 +202,29 @@ var _ = Describe("Kubectl client", func() {
 		})
 	})
 
+	Describe("Kubectl run", func() {
+		It("should run a particular image on the cluster", func() {
+			nsFlag := fmt.Sprintf("--namespace=%v", ns)
+			image := "nginx"
+			rcName := "e2e-test-nginx-rc"
+
+			By("running the image " + image)
+			runKubectl("run", rcName, "--image="+image, nsFlag)
+			By("verifying the rc " + rcName + " was created")
+			output := runKubectl("get", "rc", nsFlag)
+			if !strings.Contains(output, rcName) {
+				Failf("Failed creating rc " + rcName)
+			}
+			By("verifying the pod controlled by rc " + rcName + " was created")
+			output = runKubectl("get", "pods", nsFlag)
```

Comment:
Actually, since we're verifying that pod was created as expected, it's probably better not to use the client instead of kubectl output.
We already have a client (line 60) so you can do
```
c.ReplicationControllers(ns).Get(rcName)
```
, verify the spec calls for 1 pod with expected image, and then do
```
label := labels.SelectorFromSet(labels.Set(map[string]string{"run": "nginx"}))
c.Pods(ns).List(label, fields.Everything())
```
and verify there is 1 pod with expected image.